Sources/Swift/Tools/SentryLogger.swift

```diff
@@ -135,4 +136,19 @@ public final class SentryLogger: NSObject {
             attributes["sentry.trace.parent_span_id"] = .string(span.spanId.sentrySpanIdString)
         }
     }
+
+    private func addUserAttributes(to attributes: inout [String: SentryLog.Attribute]) {
+        guard let user = sentry_getCurrentUser() else {
```

Comment:
`h`: Please retrieve the current user from the hub instead. We already initialize the logger with a hub. Accessing the global hub via `sentry_getCurrentUser` seems unnecessary. It's also better for testability.